Source/ARTConnection+Private.h

```diff
@@ -7,12 +7,20 @@ NS_ASSUME_NONNULL_BEGIN
 
 @class ARTRealtimeInternal;
 
+@interface ARTConnectionRecoveryKey : NSObject
+@property (nullable, readwrite, strong, nonatomic) NSString *connectionKey;
+@property (readwrite, assign, nonatomic) int64_t msgSerial;
+@property (readwrite, strong, nonatomic) NSMutableDictionary<NSString *, NSString *> *channelSerials;
+ 
+- (nullable NSString *)jsonString;
```

Comment:
Aligned with my comment on the implementation-side of this method, I don't feel that it's appropriate that it's possible for this interface to return `nil`.